src/libs/actions/PersonalDetails.js

```diff
@@ -478,6 +514,14 @@ function clearAvatarErrors() {
     });
 }
 
+/**
+ * Get private personal details value
+ * @returns {Boolean}
```

Comment:
Let's not say Boolean here if we're just going to cast it to a Boolean when it's called.